packages/coreutils/package.json

```diff
@@ -32,8 +32,10 @@
     "watch": "tsc -b --watch"
   },
   "dependencies": {
+    "@phosphor/commands": "^1.6.1",
```

Comment:
This is a dep (instead of dev dep) because the typing is a part of the API?

Reply:
`InstanceTracker` uses `CommandRegistry`.